### DISCUSSION TITLE
quite perfect but.. left menu?

### DISCUSSION THREAD

**Discussion by StefanoCecere** [jpanther/congo#416] [General]
i don't see any question about a left side menu... how can you all deal with complex documentions, or organize sites with hundreads of pages?

the question is: are you going to develop it, or should i take care about it customizing a fork?
thank you!

**Comment by jpanther** [comment]
Thanks for the suggestion, when you say a left menu, are you referring to the table of contents on each article page, or some other menu?

**Reply by lschierer** [reply]
The left side menu provides for a section level navigation, whereas the top menu, particularly since is really just a series of links and not really *menus* is more for navigating between major sections.  That leaves the ToC on the right for page level navigation.

**Comment by StefanoCecere** [comment]
i refer to the architecture of the documents in the site.. for example i have dozens of "books" and resources..
usually we group them thematically in a top nav bar, and then on the left we organize the contents,
then each peage has it's own TOC.


se for example https://2042ed.org/scuola/

**Comment by akclace** [comment]
For docs with lots of pages, being able to show the list of sections and pages is very useful. For example, see the panel on the left at https://www.docsy.dev/docs/adding-content/navigation/ (on a larger display). Being able to do that with Congo will be great.
